assets/prototype/domain/eventEditor/ticketsList/ticketPriceCalculator/hooks/useOnSubmitPrices.js

```diff
@@ -47,8 +56,13 @@ const useOnSubmitPrices = (existingPrices) => {
 			deleteEntity({ id });
 		});
 
+		const normalizedTicketFields = {
+			...ticket,
+			price: parseFloat(ticket.price || 0),
+			reverseCalculate: !! ticket.reverseCalculate,
```

Comment:
`ticket.reverseCalculate` is observed to be a string e.g. `"false"`, `!! ticket.reverseCalculate` will always be true.

Reply:
Let's avoid using double negation trick.
More info on why:
- https://www.sitepoint.com/javascript-double-negation-trick-trouble/
- https://riptutorial.com/javascript/example/3047/double-negation----x-